Mod Menu Helper/assets/modmenu/lang/tr_tr.json

```diff
@@ -5,40 +5,45 @@
   "modmenu.summaryTranslation.advancements-enlarger": "§c*§r Daha büyük başarım ekranı\n§8✎ (seçenekleri yok)",
   "modmenu.summaryTranslation.animatica": "OptiFine'ın hareketli dokular\n§8✎ (sadece gelişmiş seçenekler)",
   "modmenu.summaryTranslation.antighost": "Hayalet blokları /ghost\n§9✎§8 tuş atamasıyla düzelt.",
```

Comment:
This line should then be changed as well

Reply:
There is 3 lines there. Which line?

Reply:
In turkish "ile" can be combined. In this case, yla is "ile" so this is correct.

Reply:
Also I found a mistake /ghost isn't a keybind it is a command.

Reply:
You added "ile düzelt" to line 8 but not line 7, that's what I mean.

Reply:
> In turkish "ile" can be combined. In this case, la is "ile" so this is correct. y is combine letter.

"düzelt" is already there. "ile" was explained in quote.

Reply:
No, en_us is right.

This appears in-game like this:
> Hayalet blokları /ghost
> ✎ tuş atamasıyla düzelt.

The first line should say what the mod does and second line should say what options it has - this mod has an optional keybind.

The first line should be the same in Mod Menu and pause menu screen, that's why there are two rows with identical text.

Reply:
In Turkish, verbs are at the end but in English it isn't. That's why it creates a problem.

Reply:
You still don't get it 😄

The first line should say "Fix ghost blocks with [the command] /ghost". 
The second line should say "[This mod has an optional] keybind". 

(Note that the words in [] can be added or not depending on whether it is needed for your language and whether it fits in-game.)

Since the first line has the same meaning on both Mod Menu and pause menu, it should be  identical on both translation strings.

Reply:
So on second line it says keybind of mod

Reply:
I think I fixed it.